packages/dashmate/docs/installation.md

```
@@ -0,0 +1,77 @@
+Once Node.JS has been installed, use NPM to install dashmate:
+
+```bash
+npm install -g dashmate
+```
+
+## Verifying Installation
+
+After installation, verify that Dashmate is correctly installed by running:
+
+```bash
+$ dashmate --version
```

markdownlint-cli2 (0.17.2)

Dollar signs used before commands without showing output
null

(MD014, commands-show-output)